src/status_im/contexts/settings/wallet/saved_addresses/view.cljs

```diff
@@ -36,6 +38,10 @@
        {:title               (i18n/label :t/saved-addresses)
         :accessibility-label :saved-addresses-header
         :right               :action
+        :on-press            (fn []
```

Comment:
you should probably declare this outside, you can check if it triggers a re-render of the overlay if the sub `wallet/saved-addresses` changes (I believe it will, but if you declare it outside, it should not)